lib/formatAtRules.js

```diff
@@ -111,6 +111,10 @@ function formatAtRules (root, params) {
       atrule.raws.between = ''
     }
 
+    if (atrule.name === 'value') {
```

Comment:
Could you add this to https://github.com/morishitter/stylefmt/pull/258/files#diff-2e3695da4b32fe872af0eb57a364734dR83 ?